manim/scene/scene.py

```diff
@@ -61,13 +61,14 @@ def construct(self):
 
     def __init__(self, renderer=None, **kwargs):
         Container.__init__(self, **kwargs)
-        if renderer is None:
-            self.renderer = CairoRenderer(
+        self.renderer = (
+            renderer
+            or getattr(self, "renderer", None)
```

Comment:
Well yeah. Because I wore my big boy pants and used a metaclass to handle the switch to the specialized renderer. I could make a bookmark to have livestreaming set to False so when it is true the default renderer will use the special file writer I made.